asio/cmake/install-config.cmake

```diff
@@ -0,0 +1,4 @@
+include(CMakeFindDependencyMacro)
+find_dependency(Threads)
```

Comment:
needed to add this to make the config package usable

Reply:
Yes, this is covered by the templates generated using a package manager and this example: https://github.com/friendlyanon/cmake-init-transitive